src/App.tsx

```diff
@@ -1,54 +1,77 @@
-import React from 'react';
+import React, { useRef, useState } from 'react';
+import cn from 'classnames';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+  const [filteredPeople, setFilterdPeople] = useState(peopleFromServer);
```

Comment:
```suggestion
  const [filteredPeople, setFilteredPeople] = useState(peopleFromServer);
```
It can be just a common variable